ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.ts

```diff
@@ -317,7 +317,7 @@ describe('Workload CVE Image Single page', () => {
     });
 
     describe('SBOM generation tests', () => {
-        const sbomGenerationButtonSelector = 'button:contains("Generate SBOM")';
+        const sbomGenerationButton = 'button:contains("Generate SBOM")';
```

Comment:
Looking at this with fresh eyes, it is more readable to "cy.get" the _button_, not the _selector_.